README.rst

```diff
@@ -1,9 +1,27 @@
 |logo| Pootle
 =============
 
-|chat| |build| |health| |coverage| |requirements|
+.. image:: https://img.shields.io/gitter/room/translate/pootle.svg?style=flat-square
```

Comment:
My goal in the fixes to the README where:

1. Its readable by humans
2. Its nice in github
3. It works in pypi (but that I was never able to test)

Just flagging that so that we actually try keep those in mind with any of these changes.